exercises/darts/README.md

```diff
@@ -1,20 +1,22 @@
 # Darts
 
-Write a function that returns the earned points in a single toss of a Darts game.
+Write a method that returns the earned points in a single toss of a Darts game.
 
 [Darts](https://en.wikipedia.org/wiki/Darts) is a game where players
 throw darts to a [target](https://en.wikipedia.org/wiki/Darts#/media/File:Darts_in_a_dartboard.jpg).
 
-In our particular instance of the game, the target rewards with 4 different amounts of points, depending on where the dart lands:
+In our particular instance of the game, the target rewards with 4 different amounts of points, depending on where the dart lands.
 
-* If the dart lands outside the target, player earns no points (0 points).
-* If the dart lands in the outer circle of the target, player earns 1 point.
-* If the dart lands in the middle circle of the target, player earns 5 points.
-* If the dart lands in the inner circle of the target, player earns 10 points.
+If the dart lands:
 
-The outer circle has a radius of 10 units (This is equivalent to the total radius for the entire target), the middle circle a radius of 5 units, and the inner circle a radius of 1. Of course, they are all centered to the same point (That is, the circles are [concentric](http://mathworld.wolfram.com/ConcentricCircles.html)) defined by the coordinates (0, 0).
+* outside the target: 0 points.
+* in the outer circle of the target: 1 point.
+* in the middle circle of the target: 5 points.
+* in the inner circle of the target: 10 points.
 
-Write a function that given a point in the target (defined by its `real` cartesian coordinates `x` and `y`), returns the correct amount earned by a dart landing in that point.
+The outer circle has a radius of 10 units (This is equivalent to the total radius for the entire target), the middle circle a radius of 5 units, and the inner circle a radius of 1. They are all centered to the same point (That is, the circles are [concentric](http://mathworld.wolfram.com/ConcentricCircles.html)) defined by the coordinates (0, 0).
+
+Write a method that, given a point in the target (defined by its `real` Cartesian coordinates `x` and `y`), returns the correct amount earned by a dart landing in that point.
```

Comment:
Should `real` be marked up as code?  Or is this meant to be quoted?  And how does this compare with pretend Cartesian coordinates?

Reply:
I noted that is in the canonical source as well.  Other than that, I think it looks good.  Let me know what you think, I am not opposed to bringing it in.